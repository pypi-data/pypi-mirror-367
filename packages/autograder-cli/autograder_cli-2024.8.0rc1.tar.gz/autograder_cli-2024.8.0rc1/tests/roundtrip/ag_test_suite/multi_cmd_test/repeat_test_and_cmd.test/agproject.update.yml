project:
  name: Test Project ce74d0f7c3f54a8286d8bb70a911513b
  timezone: America/Chicago
  course:
    name: Test Course
    semester: Summer
    year: 2014
  settings:
  instructor_files:
  - local_path: input1.txt
  - local_path: input2.txt
  - local_path: output1.txt
  - local_path: output2.txt
  - local_path: stderr1.txt
  - local_path: stderr2.txt
  test_suites:
  - name: Suite
    test_cases:
    - name: Test test_id
      commands:
      - name: $stage
        cmd: $script test_id
        stdin:
          source: instructor_file
          instructor_file: $input
        stdout:
          compare_with: instructor_file
          instructor_file: $stdout
        stderr:
          compare_with: instructor_file
          instructor_file: $stderr
        repeat:
        - $stage: Compile
          $script: bash compile.sh
        - $stage: Run
          $script: bash run.sh

      repeat:
      - test_id: 42
        $input: input2.txt
        $stdout: output2.txt
        $stderr: stderr2.txt
      - test_id: 43
        $input: input1.txt
        $stdout: output1.txt
        $stderr: stderr1.txt
