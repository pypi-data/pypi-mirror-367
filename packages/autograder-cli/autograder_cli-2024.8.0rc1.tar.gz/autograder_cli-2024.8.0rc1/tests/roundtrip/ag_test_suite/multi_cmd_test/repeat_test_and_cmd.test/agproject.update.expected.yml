project:
  name: Test Project ce74d0f7c3f54a8286d8bb70a911513b
  timezone: America/Chicago
  course:
    name: Test Course
    semester: Summer
    year: 2014
  settings: {}
  instructor_files:
  - local_path: input1.txt
  - local_path: input2.txt
  - local_path: output1.txt
  - local_path: output2.txt
  - local_path: stderr1.txt
  - local_path: stderr2.txt
  test_suites:
  - name: Suite
    test_cases:
    - name: Test 42
      commands:
      - name: Compile
        cmd: bash compile.sh 42
        stdout:
          compare_with: instructor_file
          instructor_file: output2.txt
        stderr:
          compare_with: instructor_file
          instructor_file: stderr2.txt
      - name: Run
        cmd: bash run.sh 42
        stdout:
          compare_with: instructor_file
          instructor_file: output2.txt
        stderr:
          compare_with: instructor_file
          instructor_file: stderr2.txt
    - name: Test 43
      commands:
      - name: Compile
        cmd: bash compile.sh 43
        stdout:
          compare_with: instructor_file
          instructor_file: output1.txt
        stderr:
          compare_with: instructor_file
          instructor_file: stderr1.txt
      - name: Run
        cmd: bash run.sh 43
        stdout:
          compare_with: instructor_file
          instructor_file: output1.txt
        stderr:
          compare_with: instructor_file
          instructor_file: stderr1.txt
